HackIllinois/Assets.xcassets/Chicken.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Chicken-4.png",
```

Comment:
Make sure all image files for the chicken following the same naming convention. This project uses the ...@1x, ...@2x, ...@3x. So for the chicken, it would be Chicken@1x, Chicken@2x, Chicken@3x.